spec/models/user_spec.rb

```diff
@@ -25,30 +57,12 @@
     expect(user).to_not be_valid
   end
 
-  it 'is not valid without a gender' do
-    user = @user
-    user.gender = nil
-    expect(user).to_not be_valid
-  end
-
-  # it 'is not valid if gender has a value not valid' do
-  #   user = User.new(email: 'test@test.com', gender: 'X', password: 'ABC12345',
-  #   password_confirmation: 'ABC12345')
-  #   expect(user).to raise_error(ArgumentError, /'X' is not a valid gender/)
-  # end
-
   it 'is not valid without a password' do
     user = @user
```

Comment:
Here you can build a user object once and use it in every test.

See https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let
```suggestion
let(:user) { build :user }
```